### PR TITLE
Update preemptive troubleshooting

### PR DESCRIPTION
**Pull request by yellowjaguar5**
- New diagrams
- Update image widths
- Text/title updates

These changes were split from #200.

### CODE REVIEW

File: source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst [hunk 1]
```diff
@@ -7,98 +7,109 @@ Robot Preemptive Troubleshooting
...
 
 Apply considerable force onto the battery cable at 90 degrees to try to move the direction of the cable leaving the battery, if successful the connection was not tight enough to begin with and it should be redone.
 
-Secure the battery to robot connection
-------------------------------------------------------------------
+Securing the Battery to the Robot
+---------------------------------
 
+.. image:: images/preemptive-troubleshooting/fallen-battery.jpg
+   :width: 49%
```
Comment (Daltz333) [inline source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst:26]: Please don't use percentages, they break mobile

File: source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst [hunk 2]
```diff
@@ -7,98 +7,109 @@ Robot Preemptive Troubleshooting
...
+
 If you can remove the blade fuses by hand then they are not in completely. Make sure that they are completely seated in the PDP so that they don't pop out during robot operation.
 
 roboRIO swarf
-------------------------------------
+-------------
 
 Swarf is: fine chips or filings of stone, metal, or other material produced by a machining operation. Often modifications must be made to a robot while the control system parts are in place. The circuit board for the roboRIO is conformally coated, but that doesn't absolutely guarantee that metal chips won't short out traces or components inside the case. In this case, you must exercise care in making sure that none of the chips end up in the roboRIO or any of the other components. In particular, the exposed 3 pin headers are a place where chips can enter the case. A quick sweep through each of the four sides with a flashlight is usually sufficient to find the really bad areas of infiltration.
```
Comment (Daltz333) [inline source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst:82]:
```suggestion
Swarf is fine chips or filings of stone, metal, or other material produced by a machining operation. Often modifications must be made to a robot while the control system parts are in place. The circuit board for the roboRIO is conformally coated, but that doesn't absolutely guarantee that metal chips won't short out traces or components inside the case. In this case, you must exercise care in making sure that none of the chips end up in the roboRIO or any of the other components. In particular, the exposed 3 pin headers are a place where chips can enter the case. A quick sweep through each of the four sides with a flashlight is usually sufficient to find the really bad areas of infiltration.
```

File: source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst [hunk 3]
```diff
@@ -7,98 +7,109 @@ Robot Preemptive Troubleshooting
...
-Reproducing problems in the pit
-------------------------------------
+Reproducing Problems in the Pit
+-------------------------------
 
-Beyond the normal shaking and rattling of all cables while the robot is power and tethered, you might try picking up one side of the robot  off the ground and drop it, and see if you lose connection. The driving on the field, especially when trying to breach defenses will often be very violent. It's better to see it fail in the pit rather than in a critical match.
+Beyond the normal shaking of cables whilst the robot is powered and tethered, it is suggested that one side of the robot be picked up and dropped. Driving on the field, especially against defenders, will often be very violent, and this helps makes sure nothing falls out. It is better for the robot to fail in the pits rather than in the middle of a match.
 
 When doing this test it’s important to be ethernet tethered and not USB tethered, otherwise you are not testing all of the critical paths.
 Check firmware and versions
```
Comment (Daltz333) [inline source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst:107]: >Check firmware and versions
I believe this should be a section title

File: source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst [hunk 4]
```diff
@@ -7,98 +7,111 @@ Robot Preemptive Troubleshooting
...
 Wiggle battery harness connector. Often these are loose because the screws loosen, or sometimes the crimp is not completely closed.  You will only catch the really bad ones though because often the electrical tape stiffens the connection to a point where it feels stiff. Using a voltmeter or Battery Beak will help with this.
 
 Apply considerable force onto the battery cable at 90 degrees to try to move the direction of the cable leaving the battery, if successful the connection was not tight enough to begin with and it should be redone.
 
-Secure the battery to robot connection
-------------------------------------------------------------------
+Securing the Battery to the Robot
+---------------------------------
 
+.. image:: images/preemptive-troubleshooting/fallen-battery.jpg
```
[the remark this answers is not in the document]
Comment (yellowjaguar5) [reply source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst:25]: I'm not sure about rights and after thinking about it more, it doesn't feel good to call out another team. Up to you. I can remove the team number from the bumper if that helps.

File: source/docs/hardware/hardware-basics/images/preemptive-troubleshooting/tighten-breaker.svg [hunk 5]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (yellowjaguar5) [reply source/docs/hardware/hardware-basics/images/preemptive-troubleshooting/tighten-breaker.svg]: Fixed in f4edaa8.

File: source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst [hunk 6]
```diff
@@ -7,98 +7,111 @@ Robot Preemptive Troubleshooting
...
     In *FIRST*\ |reg| Robotics Competition, robots take a lot of stress while driving around the field. It is important to make sure that connections are tight, parts are bolted securely in place and that everything is mounted so that a robot bouncing around the field does not break.
 
-Check battery connections
-------------------------------
+Check Battery Connections
+-------------------------
 
-.. image:: images/preemptive-troubleshooting/preCheckBatt.png
-   :width: 400
+.. image:: images/preemptive-troubleshooting/battery-bolt-check.svg
```
[the remark this answers is not in the document]
Comment (yellowjaguar5) [reply source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst]: Fixed in f4edaa8.

File: source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (AustinShalit) [reply source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst:25]: Let's remove it

[the remark this answers is not in the document]
Comment (Daltz333) [reply source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst:25]: I can get a fallen battery image if wanted (take one of my team)

**Pull request merged**